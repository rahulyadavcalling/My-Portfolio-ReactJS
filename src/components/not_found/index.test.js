import React from "react";
import renderer from "react-test-renderer";

import NotFound from "./index.js";

it("renders correctly", () => {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
