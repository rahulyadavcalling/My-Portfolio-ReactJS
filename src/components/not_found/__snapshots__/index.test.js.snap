// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="sc-bdVaJa fhlziI"
>
  <h1
    className="sc-bwzfXH hlfBfo"
  >
    404 
    <br />
    Page not found.
  </h1>
</div>
`;
